esm-wrapper.mjs

```diff
@@ -0,0 +1,145 @@
+// just pre-load all the stuff that index.js lazily exports
+import {createRequire} from 'module';
+
+const require = createRequire(import.meta.url);
+
+const internalRe = require('./internal/re');
+
+const re = internalRe.re;
+const src = internalRe.src;
+const tokens = internalRe.t;
+const SEMVER_SPEC_VERSION = require('./internal/constants').SEMVER_SPEC_VERSION;
```

Comment:
And similarly, `import { SEMVER_SPEC_VERSION } from './internal/constants'` etc.